# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/sparc-bus-26headers.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- xc/programs/Xserver/hw/xfree86/os-support/bus/xf86Sbus.h.orig	2004-08-22 23:15:10.000000000 +0200
+++ xc/programs/Xserver/hw/xfree86/os-support/bus/xf86Sbus.h	2004-08-22 23:15:25.000000000 +0200
@@ -27,6 +27,7 @@
 
 #if defined(linux)
 #include <asm/types.h>
+#include <linux/fb.h>
 #include <asm/fbio.h>
 #include <asm/openpromio.h>
 #elif defined(SVR4)
